src/config/partners/coke.ts

```diff
@@ -125,6 +139,8 @@ const cokeConfig: Partner = {
       creator: '0x41051B6f8397435C32915C40B18809118F2E9077',
       image: '/partners/coke/drop/vangogh.jpg',
       type: 'erc-721',
+      interactWithNFTLink:
+        'https://nft.coinbase.com/0x32cF27F2753e90948195b64EC55C486EE640cF61',
```

Comment:
I think rather than hardcoding this everywhere we could just make it default to `https://nft.coinbase.com/${address}`

Reply:
this doesn't seem to match the address above also @lauchness

Reply:
also I think this is the correct URL format `https://nft.coinbase.com/collection/base/{address}`

Reply:
if we construct this link in the code rather than hard coding than we don't have to worry about wrong address